sklearn/preprocessing/data.py

```diff
@@ -2865,51 +2866,60 @@ def power_transform(X, method='box-cox', standardize=True, copy=True):
     Parameters
     ----------
     X : array-like, shape (n_samples, n_features)
-        The data to be transformed using a power transformation.
+        The data used to estimate the optimal transformation parameters.
```

Comment:
I'd leave the original version: ultimately the goal of `power_transform()` is to transform X, the estimation of lambda is just an intermediary step.